apps/asap-server/src/controllers/news.ts

```diff
@@ -0,0 +1,43 @@
+import { ListNewsResponse, NewsResponse } from '@asap-hub/model';
+import { RestNews } from '@asap-hub/squidex';
+
+import { InstrumentedSquidex } from '../utils/instrumented-client';
+import { parseNews } from '../entities';
+
+export default class News implements NewsController {
+  news: InstrumentedSquidex<RestNews>;
+
+  constructor(ctxHeaders?: Record<string, string>) {
+    this.news = new InstrumentedSquidex('news-and-events', ctxHeaders);
```

Comment:
the ticket says to update squidex - dont we want to update the entity? or is that coming in another PR?

Reply:
Its relabled in squidex and I was concerned if I renamed it here it would drop the data